java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -3464,6 +3681,16 @@ void testRangeWindowingLead() {
         .column(1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1) // GBY Key
         .column(0, 0, 0, 0, 1, 1, 1, 1, 2, 2, 2, 2, 2) // GBY Key
         .column(7, 5, 1, 9, 7, 9, 8, 2, 8, 0, 6, 6, 8) // Agg Column
+        .column(new ListType(false, new BasicType(true, DType.INT32)),
```

Comment:
Not a review suggestion. Just FYI: I think SparkSQL doesn't support calling `LEAD()`/`LAG()` on `RANGE` windows. It might be worth checking. This path might simply not get exercised from the `spark-rapids` plugin.

Reply:
Done

Reply:
Removed the lead/lag for range window test